mbed_tools_ci/generate_news.py

```diff
@@ -43,7 +43,7 @@ def _calculate_version(commit_type: CommitType, use_news_files: bool) -> \
         use_news_files: Should the version be dependant on changes recorded in news files
 
     Returns:
-        A semver-style version for the latest release
+        (is new version, A semver-style version for the latest release)
```

Comment:
Is this docstring correct? I have no idea how to document tuple return types. @rosiemiller2 ?

Reply:
flake8 did not complain so I assumed so. Could not find a better answer

Reply:
FWIW https://stackoverflow.com/questions/29221551/can-sphinx-napoleon-document-function-returning-multiple-arguments - I know it's sphinx, but they mention google style.